src/index.js

```diff
@@ -1,84 +1,70 @@
-import {
-  generateColorExtension,
-  cgColorString,
-  generateFontExtension,
-  options,
-  linearGradientLayer,
-  radialGradientLayer,
-} from './helpers';
+import Zepcode from './zepcode';
 
 function styleguideColors(context, colors) {
-  return generateColorExtension(colors, options(context));
+  return new Zepcode(context).generateColorExtension(colors);
 }
 
 function styleguideTextStyles(context, textStyles) {
-  return generateFontExtension(textStyles);
+  return Zepcode.generateFontExtension(textStyles);
 }
 
 function layer(context, layerParams) {
+  const zepcode = new Zepcode(context);
   let string = '';
+  const newlineBeforeContent = () => (string.length ? '\n\n' : '');
+
   if (layerParams.fills.length) {
     const { gradient } = layerParams.fills[0];
+    let gradientString = '';
+
     if (gradient !== undefined) {
       switch (gradient.type) {
         case 'linear':
-          string += linearGradientLayer(
-            gradient,
-            context.project,
-            options(context)
-          );
+          gradientString = zepcode.linearGradientLayer(gradient);
           break;
         case 'radial':
-          string += radialGradientLayer(
-            gradient,
-            context.project,
-            options(context)
-          );
+          gradientString = zepcode.radialGradientLayer(gradient);
           break;
         default:
           break;
       }
     }
+    string += gradientString;
   }
+
   if (layerParams.opacity !== 1) {
-    if (string.length > 0) {
-      string += '\n\n';
-    }
-    string += `view.alpha = ${layerParams.opacity.toFixed(2)}\n`;
+    string += `${newlineBeforeContent()}view.alpha = ${layerParams.opacity.toFixed(
+      2
+    )}\n`;
   }
-  if (layerParams.borders.length > 0) {
+
+  if (layerParams.borders.length) {
     const border = layerParams.borders[0];
```

Comment:
Is it a correct check of non-zero values in JS?

Reply:
Yep, the result will be same, but second way in short